builder/dockerfile/parser/testfiles/escape-nonewline/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# escape = ``
```

Comment:
@jhowardmsft It would be very easy to fix but I don't agree this is correct. Directive syntax should be `# key=value`, not `# key=value+any garbage`. Directives in general are not limited to single char.